fvm/evm/stdlib/contract.cdc

```diff
@@ -183,12 +242,27 @@ contract EVM {
 
     /// Runs an a RLP-encoded EVM transaction, deducts the gas fees,
     /// and deposits the gas fees into the provided coinbase address.
-    ///
-    /// Returns true if the transaction was successful,
-    /// and returns false otherwise
     access(all)
-    fun run(tx: [UInt8], coinbase: EVMAddress) {
-        InternalEVM.run(tx: tx, coinbase: coinbase.bytes)
+    fun run(tx: [UInt8], coinbase: EVMAddress): Result {
+        return InternalEVM.run(
+                tx: tx, 
+                coinbase: coinbase.bytes
+        ) as! Result
```

Comment:
Same as for call: Can the return type of `InternalEVM.run` be `Result`?

Reply:
`internalEVMTypeRunFunctionType` is still `Bool`

Reply:
fixed the internalEVMTypeRunFunctionType.

Reply:
Not possible to use `EVM.Result` in `InternalEVM.run`, added a comment